handlers.c

```diff
@@ -745,17 +745,25 @@ bool handler__lregions(globals_t * vars, char **argv, unsigned argc)
     return true;
 }
 
+static bool do_search ( scan_match_type_t m, globals_t * vars, uservalue_t * pval, char ** argv, unsigned argc );
+static bool try_get_range_values ( const char * arg, uservalue_t ** pval );
+
 /* the name of the function is for history reason, now GREATERTHAN & LESSTHAN are also handled by this function */
 bool handler__decinc(globals_t * vars, char **argv, unsigned argc)
 {
     uservalue_t val;
-    scan_match_type_t m;
+    uservalue_t * pval = &val;
```

Comment:
Don't introduce new pointer variables for an existing variable which is used in a similar way at other locations if not changing that everywhere. That `uservalue_t val` stuff is used at multiple places, you are touching unrelated code parts, so please don't introduce that `pval` variable.